packages/react-core/src/components/ContextSelector/ContextSelectorToggle.tsx

```diff
@@ -10,8 +10,9 @@ export interface ContextSelectorToggleProps {
   id: string;
   /** Classes applied to root element of toggle */
   className?: string;
-  /** Text that appears in the Context Selector Toggle */
-  toggleText?: string;
+  /** Component or string that will be used in the Context Selector Toggle */
+  /** This prop is placed in a button */
```

Comment:
This is great! 
Let's also be just a bit more explicit - maybe something like `This prop is placed in a button, and cannot contain any interactive elements`

And maybe both prop descriptions for `toggleText` should say the same thing. Could you update the description in `ContextSelector.tsx` as well?

Reply:
Oops! My bad, I had missed updating it in the `ContextSelector.tsx`, thanks for pointing it out @nicolethoen :)

Reply:
Updated the description with the suggestion as well 🚀